resources/src/trade.rs

```diff
@@ -299,3 +299,100 @@ mod trade_tests {
         );
     }
 }
+
+/// The aggregation of trades for a specifc base/counter pair of assets over a given
+/// time period.
+#[derive(Deserialize, Serialize, Debug)]
+pub struct TradeAggregation {
+    // Several fields are omitted since they don't seem to be in the actual response from horizon.
+    // Or they don't seem to make sense.
+    //
+    // The rational versions of the prices also only seem to be present in the examples. When
+    // querying the actual horizon API they were absent. They probably don't contain much value
+    // over the "amount" field anyhow so I left them off.
+    timestamp: u64,
```

Comment:
I know the endpoints return different things, but should we convert this into a DateTime to match the format in Trade?